common/crypto/platform/openssl/crypto.cpp

```diff
@@ -472,11 +571,17 @@ expected::ExpectedBool VerifySignData(
 	ret = EVP_PKEY_verify(
 		pkey_signer_ctx.get(), signature.data(), signature.size(), shasum.data(), shasum.size());
 	if (ret < 0) {
```

Comment:
Pre-existing, and basically everywhere in the file, but shouldn't these be:
```suggestion
	if (ret < OPENSSL_SUCCESS) {
```
?

Reply:
No, `OPENSSL_SUCCESS` is `1`.

ret is an error if `< 0`. Verification failed if `0`, and succeeded if `1`

But this should obviously be communicated better.